### ISSUE TITLE
Problem z pobraniem listu przewozowego

### ISSUE THREAD

**Issue by aszymkowiak** [allegro/allegro-api#9780]
Występuje problem: 
 - [x] z zasobem REST API / with a REST API resource
 - [x] ze środowiskiem produkcyjnym / with the production environment
 - [ ] ze środowiskiem testowym / with the test environment
 - [ ] z logowaniem lub autoryzacją / with login or authorization
 - [ ] na portalu https://developer.allegro.pl/ / at https://developer.allegro.pl/en/
 - [ ] inny / other

Dzień dobry,
przy próbie pobrania listu przewozowego otrzymuję błąd:
- Carrier Api Client is temporary unavailable - try again later or report problem(TraceId: 6ca1e06e5160ca83)
- ShipX token provided in WZA account settings is missing or invalid (TraceId: 36600ae67c4ae30e)
Pierwszy błąd jest dla mnie jasny i zakładam, że sprawa rozwiąże się w najbliższym czasie. Mam pytanie co do drugiego błędu, ponieważ nie modyfikowałem ostatnio połączenia z API. Jest to związane z wprowadzonymi zmianami, czy po prostu chwilowy błąd (tak jak pierwszy)?
Z góry dziękuję za odpowiedź.

**Comment by MartaNowaczyk** [comment]
Tak, pierwsza może się zdarzyć i raczej wystarczy poczekać i ponowić, jednak gdyby tak się nei stało proszę o informację.
W drugim przypadku sprawdź czy nie były edytowane ustawienia, bądź czy nie zmieniły się dane dla ShipX.

**Comment by aszymkowiak** [comment]
Jeśli chodzi o pierwszy problem to przestał występować. Co do drugiego błędu zauważyłem, że występuje on dla jednej przesyłki. Przykład:
- 633a92ce-137a-46ea-b626-d203b8bdba3b - błąd
- 2e62a2ec-a153-4edd-9abd-e7fb20f131fb - brak błędu
Obie przesyłki mają tą samą metodę dostawy 2488f7b7-5d1c-4d65-b85c-4cbcf253fd93 i różnią się tylko gabarytem. Jaki może być tego powód?
